cli/src/main/scala/com/codacy/analysis/cli/CLIError.scala

```diff
@@ -10,17 +10,17 @@ sealed trait CLIError {
 
 object CLIError {
 
-  def from(coreError: Analyser.Error): CLIError = {
+  def from(coreError: AnalyserError): CLIError = {
     coreError match {
-      case Analyser.Error.ToolExecutionFailure(toolType, toolName) =>
+      case AnalyserError.ToolExecutionFailure(toolType, toolName) =>
         CLIError.ToolExecutionFailure(toolType, toolName)
-      case Analyser.Error.ToolNeedsNetwork(toolName) =>
+      case AnalyserError.ToolNeedsNetwork(toolName) =>
         CLIError.ToolNeedsNetwork(toolName)
-      case Analyser.Error.NonExistingToolInput(toolName, _) =>
+      case AnalyserError.NonExistingToolInput(toolName) =>
         CLIError.NonExistingToolInput(toolName)
-      case Analyser.Error.NoActiveToolInConfiguration =>
+      case AnalyserError.NoActiveToolInConfiguration =>
         CLIError.NoActiveToolInConfiguration
-      case Analyser.Error.NoToolsFoundForFiles =>
+      case AnalyserError.NoToolsFoundForFiles =>
         CLIError.NoToolsFoundForFiles
     }
```

Comment:
When I compiled locally I noticed that there was a warning saying that not all the cases were managed. Can you ensure the warning isn't more there or fix it?

Reply:
Done ✅